doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1814 @@
+is ever used. The security implications are believed to be minimal, but
+this issue was never fixed for backwards compatibility reasons. 
+
+=item AES_encrypt, AES_decrypt, AES_set_encrypt_key, AES_set_decrypt_key,
+AES_cbc_encrypt, AES_cfb128_encrypt, AES_cfb1_encrypt, AES_cfb8_encrypt,
+AES_ecb_encrypt and AES_ofb128_encrypt
+
+=item AES_unwrap_key, AES_wrap_key
+
+See L</Deprecated low-level encryption functions>
+
+=item AES_options
+
+There is no replacement. It returned a string indicating if the AES code was unrolled.
+
+=item ASN1_digest, ASN1_sign and ASN1_verify
+
+There is no replacement. These old functions are not used, and could be disabled
+with the macro NO_ASN1_OLD since OpenSSL 0.9.7.
+
+=item ASN1_STRING_length_set
+
+Use L<ASN1_STRING_set(3)> or L<ASN1_STRING_set0(3)> instead.
+This was a potentially unsafe function that could change the bounds of a
+previously passed in pointer.
+
+=item BF_encrypt, BF_decrypt, BF_set_key, BF_cbc_encrypt, BF_cfb64_encrypt,
+BF_ecb_encrypt and BF_ofb64_encrypt
+
+See L</Deprecated low-level encryption functions>.
+The Blowfish algorithm has been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item BF_options
+
+There is no replacement. This option returned a constant string.
+
+=item BN_is_prime_ex and BN_is_prime_fasttest_ex
+
+Use L<BN_check_prime(3)> which that avoids possible misuse and always uses at least
+64 rounds of the Miller-Rabin primality test.
+
+=item BN_pseudo_rand and BN_pseudo_rand_range
+
+Use L<BN_rand(3)> and L<BN_rand_range(3)>.
+
+=item BN_X931_derive_prime_ex, BN_X931_generate_prime_ex and BN_X931_generate_Xpq
+
+There is no replacement for these low-level functions. They were used internally
+by RSA_X931_derive_ex() and RSA_X931_generate_key_ex() which are also deprecated.
+Use L<EVP_PKEY_keygen(3)> instead.
+
+=item Camellia_encrypt, Camellia_decrypt, Camellia_set_key,
+Camellia_cbc_encrypt, Camellia_cfb128_encrypt, Camellia_cfb1_encrypt,
+Camellia_cfb8_encrypt, Camellia_ctr128_encrypt, Camellia_ecb_encrypt and
+Camellia_ofb128_encrypt.
+
+See L</Deprecated low-level encryption functions>.
+
+=item CAST_encrypt, CAST_decrypt, CAST_set_key, CAST_cbc_encrypt,
+CAST_cfb64_encrypt, CAST_ecb_encrypt and CAST_ofb64_encrypt
+
+See L</Deprecated low-level encryption functions>.
+The CAST algorithm has been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item CMAC_CTX_new, CMAC_CTX_cleanup, CMAC_CTX_copy, CMAC_CTX_free and
+CMAC_CTX_get0_cipher_ctx
+
+See L</Deprecated low-level MAC functions>.
+
+=item CMAC_Init, CMAC_Update, CMAC_Final and CMAC_resume.
+
+See L</Deprecated low-level MAC functions>.
+
+=item CRYPTO_mem_ctrl, CRYPTO_mem_debug_free, CRYPTO_mem_debug_malloc,
+CRYPTO_mem_debug_pop, CRYPTO_mem_debug_push, CRYPTO_mem_debug_realloc,
+CRYPTO_mem_leaks, CRYPTO_mem_leaks_cb, CRYPTO_mem_leaks_fp and
+CRYPTO_set_mem_debug
+
+Memory-leak checking has been deprecated in favor of clang's memory and leak
```

Comment:
It's not just clang.  What about _deprecated in favour of modern development tools._  If an example is needed: _Such as compiler memory and leak sanitizers or Valgrind._

Reply:
Done